### PR TITLE
chore (refs DPLAN-11545): Add missing Hooks/Selectors.

### PR DESCRIPTION
**Pull request by ahmad-demos**
### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-11545/Hooks-Bobsh-Platform-Dropdown

- Add missing Hooks/Selectors.

### CODE REVIEW

File: templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_header.html.twig [hunk 1]
```diff
@@ -38,6 +39,7 @@
                                         class="c-actionmenu__menuitem whitespace-normal"
                                         data-actionmenu-menuitem
                                         data-actionmenu-current
+                                        :data-cy="`baseHeader:{{ project.name | wysiwyg }}`"
```
Comment (salisdemos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_header.html.twig:42]:
```suggestion
                                        :data-cy="`baseHeader:platformSwitcher:{{ index }}`"
```
We handle external links here. I think the idx is sufficient. And using a title leads to exposed configuration-details in the tests

File: templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_oeb_header.html.twig [hunk 2]
```diff
@@ -38,6 +39,7 @@
                                         class="c-actionmenu__menuitem whitespace-normal"
                                         data-actionmenu-menuitem
                                         data-actionmenu-current
+                                        :data-cy="`baseHeader:{{ project.name | wysiwyg }}`"
```
Comment (salisdemos) [inline templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_oeb_header.html.twig:42]: see above

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_header.html.twig (hunk 1 above)
Comment (ahmad-demos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_header.html.twig]: do you mean here `{{ loop.index }}`, right?
Done in https://github.com/demos-europe/demosplan-core/pull/3206/commits/7b4b0bf5699fbdc089c67e5d24699f7ee264100f

File: templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_oeb_header.html.twig (hunk 2 above)
Comment (ahmad-demos) [reply templates/bundles/DemosPlanCoreBundle/DemosPlanCore/includes/base_oeb_header.html.twig]: Done in https://github.com/demos-europe/demosplan-core/pull/3206/commits/7b4b0bf5699fbdc089c67e5d24699f7ee264100f

**Review by salisdemos — APPROVED** [review]

**Pull request merged**